config.template

```diff
@@ -121,6 +121,18 @@ areaNH = IceArea_timeseries/iceAreaNH_climo.nc
 areaSH = IceArea_timeseries/iceAreaSH_climo.nc
 volNH = PIOMAS/PIOMASvolume_monthly_climo.nc
 volSH = none
+concNASATeamWinNH = SSMI/NASATeam_NSIDC0051/SSMI_NASATeam_gridded_concentration_NH_jfm.interp0.5x0.5.nc
+concNASATeamSumNH = SSMI/NASATeam_NSIDC0051/SSMI_NASATeam_gridded_concentration_NH_jas.interp0.5x0.5.nc
+concNASATeamWinSH = SSMI/NASATeam_NSIDC0051/SSMI_NASATeam_gridded_concentration_SH_djf.interp0.5x0.5.nc
+concNASATeamSumSH = SSMI/NASATeam_NSIDC0051/SSMI_NASATeam_gridded_concentration_SH_jja.interp0.5x0.5.nc
+concBootstrpWinNH = SSMI/Bootstrap_NSIDC0079/SSMI_Bootstrap_gridded_concentration_NH_jfm.interp0.5x0.5.nc
+concBootstrpSumNH = SSMI/Bootstrap_NSIDC0079/SSMI_Bootstrap_gridded_concentration_NH_jas.interp0.5x0.5.nc
+concBootstrpWinSH = SSMI/Bootstrap_NSIDC0079/SSMI_Bootstrap_gridded_concentration_SH_djf.interp0.5x0.5.nc
+concBootstrpSumSH = SSMI/Bootstrap_NSIDC0079/SSMI_Bootstrap_gridded_concentration_SH_jja.interp0.5x0.5.nc
+thickonNH = ICESat/ICESat_gridded_mean_thickness_NH_on.interp0.5x0.5.nc
+thickfmNH = ICESat/ICESat_gridded_mean_thickness_NH_fm.interp0.5x0.5.nc
+thickonSH = ICESat/ICESat_gridded_mean_thickness_SH_on.interp0.5x0.5.nc
+thickfmSH = ICESat/ICESat_gridded_mean_thickness_SH_fm.interp0.5x0.5.nc
 
```

Comment:
So I have a couple of what may be naive questions:

1) The variable names here are not clear at all to me.  If I were approaching this for the first time, I wouldn't be sure what things like concNASATeamWinNH means.  thickonNH is even more unclear.  Perhaps something like seaIceThickness_NH_FM and seaIceConcentrationNASA_NH_JFM

2) On a more fundamental level: with the freedom to change the filename/path completely, is there a risk that a user could add a separate iceObs file that is not 0.5 x 0.5 resolution or in the appropriate format?   I'm wondering if it would make sense to have the explicit file names for the observation files in modelvsobs.py as I think these files are assumed to remain the same.  You could also then just have 3 variables in the config file.

Let me know if the comments are off base here.